src/index.tsx

```diff
@@ -34,7 +35,7 @@ interface PortalNodeBase<C extends Component<any>> {
 }
 export interface HtmlPortalNode<C extends Component<any> = Component<any>> extends PortalNodeBase<C> {
```

Comment:
In order to avoid breaking change, I merged block/inline elements into this interface.

Alternatively, I could separate into HtmlBlockPortalNode/HtmlInlinePortalNode (breaking change), or keep HtmlPortalNode same name and export HtmlInlinePortalNode in addition (non breaking change)